### DISCUSSION TITLE
Is work continuing on this?

### DISCUSSION THREAD

**Discussion by cvarrichio** [corca-ai/EVAL#49] [General]
Is work continuing on this?  I know a couple of weeks ago this seemed vastly superior to AutoGPT, but our development seems to be slowing.

**Comment by adldotori** [comment]
Hello cvarrichio. We are so grateful for the your interest in our repo. To be honest, we're not sure that what the advantage of autogpt is either. I'm curious that if you think EVAL would have been better than AutoGPT if we had continued to improve it.

By the way, we start new project, AIvilization, you can see [here](https://github.com/corca-ai/AIvilization).

**Comment by cvarrichio** [comment]
In the first couple of weeks I extensively tested AutoGPT, babyAGI, and EVAL, and as far as I could tell EVAL was leaps and bounds ahead.  None of the others completed anything in my test suite, but EVAL completed a few (mostly) successfully, including:

- Automatically documenting code in a repository
- Creating and hosting a flask web app
- Some minor web crawling scraping tasks

The code seemed higher quality as well.  However, AutoGPT has over 1400 commits and over 14,000 forks.  I fear we are swimming against the current.

**Reply by adldotori** [reply]
What do you think is the biggest reason we haven't become more like AutoGPT?

**Reply by adldotori** [reply]
Obviously, I was confident in EVAL, but it didn't get as much attention as AutoGPT, so I was worried that I was on the right track. So, I decided to start with multi-agents.

**Comment by cvarrichio** [comment]
I think it must just be marketing and media.  I found all three of you on Twitter originally, but perhaps their early posts had more followers and retweets?  Your code was an order of magnitude better than either of them. 

I will update my clones and retest each of them and let you know where they're currently at.

**Reply by adldotori** [reply]
thank you so much! It's too helpful to me.

**Comment by adldotori** [comment]
@cvarrichio do you want to join our discord and talk with us?

**Comment by cvarrichio** [comment]
@adldotori  Yes, I'd be interested in joining your discord, let me know.  Thanks

**Comment by cvarrichio** [comment]
A flurry of activity... are we back?

**Comment by adldotori** [comment]
Can you let me know your email? I'll invite you to our discord channel. 
Actually, we are implementing this. https://github.com/corca-ai/AIvilization
